pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -295,7 +299,11 @@ func (pc *PodController) configureSecondaryInterface(pod *corev1.Pod, netinfo *S
 	if netinfo.InterfaceType == sriovInterfaceType {
```

Comment:
is there a way to avoid code duplication by using a helper function to craft the cni plugin args?

Reply:
Sure. Moved the command building into a separate function whereaboutsCmdBuilder().